### DISCUSSION TITLE
Ender-3 V3 KE / SE?

### DISCUSSION THREAD

**Discussion by gitfve** [mriscoc/Ender3V2S1#1205] [General]
New Ender3 printer again, Ender-3 V3 KE / SE?
It would be a shame if the software only works for the current Ender printers. The hardware is getting cheaper all the time and the lifespan of the hardware is not long.
Any change the firmware will fit in the new Ender-3 V3?

**Comment by TheWebMachine** [comment]
Assuming Creality didn't throw an entirely different mainboard into the mix with the V3, it shouldn't take much effort to compile for it. If Creality attempted to reinvent the wheel again, who knows what's under the hood.

However, the only way to be sure is for someone *with* that printer to be available to obtain board details and test firmware builds.

**Reply by mriscoc** [reply]
It is not only the board, the display unit is also different. so the UI need to be rewritten for that screen.

**Reply by TheWebMachine** [reply]
Ah, I missed that detail on the new design. I haven't looked at the details yet; perhaps we could still swap in a DACAI or DWIN screen like we do with the V2's incompatible touch screen?
